modules/standard/CTypes.chpl

```diff
@@ -121,9 +121,10 @@ module CTypes {
 
   /*
 
-    Represents a local C pointer for the purpose of C integration. This class
-    represents the equivalent to a C language pointer. Instances of this class
-    support assignment to other instances or nil, == or != comparison with a
+    Represents a local C pointer for the purpose of C integration. This type
+    represents the equivalent to a C language pointer `eltType*`. Instances of
+    this type support assignment to other instances or ``nil``, ``==`` or ``!=``
+    comparison with a
```

Comment:
Not a big deal but it looks like this paragraph could be rebalanced with the lines below it